examples/worker_setup_advanced.js

```diff
@@ -212,12 +210,18 @@ window.onload = async function () {
       const w = mwidth / dpi * 2.54 * 10;
       const h = mheight / dpi * 2.54 * 10;
 
-      const p1 = drawpoint(0, 0, 0);
-      const p2 = drawpoint(w, 0, 0);
-      const p3 = drawpoint(w, h, 0);
-      const p4 = drawpoint(0, h, 0);
+      let p1 = drawpoint(0, 0, 0);
+      let p2 = drawpoint(w, 0, 0);
+      let p3 = drawpoint(w, h, 0);
+      let p4 = drawpoint(0, h, 0);
+
+      // this will draw a fixed red square, uncomment only for debugging...
+      /*p1 = {x:100, y: 100};
+      p2 = {x:200, y: 100};
+      p3 = {x:200, y: 200};
+      p4 = {x:100, y: 200};*/
 
-      console.log(p1)
+      console.log(p1, p2, p3, p4)
```

Comment:
the values are not in range of x y screen coords, something is wrong (form amtrix_GL_RH? from proj matrix or what?)